README.md

```diff
@@ -41,3 +41,41 @@ This software is licensed under the MIT License, allowing you to use, copy, modi
 + Include the original copyright message and the MIT License in all copies or substantial portions of the software.
 For more details, refer to the [MIT License](LICENSE) file.
 
+## Running the Application with Docker
+
+Follow these steps to run the application using Docker.
+
+#### Step 1: Pull the Docker Image
```

Comment:
Might be good to add **git clone** as first step?

`git clone https://github.com/fungover/thunder.git`

Reply:
Since we have published docker images of the project we don't need to clone the source code. docker pull is the equivalent to cloning but for docker images :).